### PR TITLE
improvement(Cash Flow Reports)

### LINKED ISSUE #3800: Improved cash flow report
Add to cash flow report the ability to view the report by analyzing third-party accounts.

### PR DESCRIPTION
**Pull request by lomamech**
- Adding Possibilty to view the detailed cashflow report

closes #3800

### CODE REVIEW

File: client/src/modules/reports/generate/cashflow/cashflow.html [hunk 1]
```diff
@@ -35,6 +35,14 @@ <h3 class="text-capitalize" translate>REPORT.CASHFLOW.TITLE</h3>
               on-change="ReportConfigCtrl.onSelectCashboxes(cashboxes)"
               required="true">
             </bh-multiple-cashbox-select>
+            <hr>
```
Comment (mbayopanda) [inline client/src/modules/reports/generate/cashflow/cashflow.html]: Could you remove this `<hr>`

**Review by mbayopanda — APPROVED** [review]
I made a tiny comment about `<hr>`, but the report is very very nice

👍 LGTM

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

bors r+

**Comment by kwilu** [comment]
Heads up @lomamech, we just merged some commits that conflict with the changes your made in this pull request! You can review this repository's [recent commits](https://github.com/IMA-WorldHealth/bhima/commits/master) to see where the conflicts occur. Please rebase your feature branch against the `upstream/master` branch and [resolve](https://zulip.readthedocs.io/en/latest/git/troubleshooting.html#recover-from-a-git-rebase-failure) your pull request's merge conflicts accordingly.

**Pull request closed without merging**